tests/test_split_command.rs

```diff
@@ -80,4 +80,99 @@ fn test_split_by_paths() {
     A file1
     A file3
     "###);
+
+    // Insert an empty commit after @- with "split ."
+    let edit_script = test_env.set_up_fake_editor();
+    std::fs::write(
+        edit_script,
+        ["dump editor0", "next invocation\n", "dump editor1"].join("\0"),
+    )
+    .unwrap();
+    let stdout = test_env.jj_cmd_success(&repo_path, &["split", "-r", "@-", "."]);
+    insta::assert_snapshot!(stdout, @r###"
+    Rebased 1 descendant commits
+    First part: 31425b568fcf (no description set)
+    Second part: af0963926ac3 (no description set)
+    Working copy now at: 28d4ec20efa9 (no description set)
+    "###);
+    insta::assert_snapshot!(
+        std::fs::read_to_string(test_env.env_root().join("editor0")).unwrap(), @r###"
+    JJ: Enter commit description for the first part (parent).
+
+    JJ: This commit contains the following changes:
+    JJ:     A file2
+
+    JJ: Lines starting with "JJ: " (like this one) will be removed.
+    "###);
+    insta::assert_snapshot!(
+        std::fs::read_to_string(test_env.env_root().join("editor1")).unwrap(), @r###"
+    JJ: Enter commit description for the second part (child).
+
+    JJ: This commit contains the following changes:
+
+    JJ: Lines starting with "JJ: " (like this one) will be removed.
+    "###);
```

Comment:
I feel like we have enough test coverage of what the editor sees that we don't need this, but I'm fine with keeping it if you think it's useful. The same applies below.

Reply:
Sure, editor content tests removed.